client/src/components/Podium/Podium.tsx

```diff
@@ -37,15 +37,17 @@ const Podium: FC<PodiumProps> = ({
         <title>{`Partie n°${game.id}`}</title>
       </Helmet>
       <div className={classes}>
-        <p className='Podium__Result'>
-          <span className='Podium__Emoji' aria-hidden='true'>
-            {isWinner ? '🏆' : '😭'}
-          </span>
-          {isWinner ? 'Vous avez gagné !' : 'Vous avez perdu !'}
-          <span className='Podium__Emoji' aria-hidden='true'>
-            {isWinner ? '🏆' : '😭'}
-          </span>
-        </p>
+        {currentPlayerId && (
+          <p className='Podium__Result'>
+            <span className='Podium__Emoji' aria-hidden='true'>
+              {isWinner ? '🏆' : '😭'}
+            </span>
+            {isWinner ? 'Vous avez gagné !' : 'Vous avez perdu !'}
+            <span className='Podium__Emoji' aria-hidden='true'>
+              {isWinner ? '🏆' : '😭'}
+            </span>
+          </p>
+        )}
```

Comment:
We should display a message if no current player eg. 'Cette partie est terminée'